content/blog/getting-started-with-the-hpe-data-services-cloud-console-powershell-sdk.md

```diff
@@ -0,0 +1,186 @@
+    Write-Host ("Response headers: {0}" -f ($_.Exception.Response.Headers | ConvertTo-Json))
+    exit(-1)
+}	
+```
+
+As you can see the Access Token, Client Id, Client Secret and the Base URL are stored in a global variable: $Configuration. The retrieved Access Token is valid for two hours and needs to be refreshed afterwards. 
+
+P﻿lease note: the $Configuration.BaseUri depends on the DSCC location. You will need to use BaseUri given for your DSCC location. At the time of writing this blog the three possible DSCC BaseUri are:
+
+* E﻿urope:  https://eu1.data.cloud.hpe.com
+* U﻿SA:  https://us1.data.cloud.hpe.com
+* A﻿PJ:   https://jp1.data.cloud.hpe.com
+
+I﻿f I want to have a list of storage systems, that I have access to on the DSCC, then I can use the *Invoke-SystemList* API endpoint as given in the following example:
+
+```powershell
+try {
+    $Systems = (Invoke-SystemsList).items
+    $SystemIds = @{}
+	foreach($s in $Systems){
+		$SystemIds += @{$s.name = $s.Id}
+	}
+} catch {
+    Write-Host ("Exception occurred when calling Invoke-DeviceType1SystemsList: {0}" -f ($_.ErrorDetails | ConvertFrom-Json))
+    Write-Host ("Response headers: {0}" -f ($_.Exception.Response.Headers | ConvertTo-Json))    
+}
+```
+
+A﻿fter getting the systems list with the *Invoke-SystemsList* command, I stored the system name and system Id in an array, because I will need the system Id for many of the possible API commands. Generally it is the case that  the systems, volume, hosts, initiators etc. you want to manipulate in your scripts are defined by their ids. I.e. if you work with the DSCC API you will work with the id of the object you want to change or delete. 
+
+If I want to create for instance a new volume, then I do need to define the volume parameters like name, size etc. and the system where I want to create the volume. The system is defined by its system id and a new volume can be created with the *Invoke-VolumeCreate* command that has two input parameters: the system id and the volume parameters that are stored in a *CreateVolumeInput* object using the *Initialize-CreateVolumeInput* method. 
+
+```powershell
+$SystemId = $System.Primera650
+$CreateVolumeInput = Initialize-CreateVolumeInput -Comments "DSCC API -Thomas Beha" `
+  -Count 1 `
+  -DataReduction $true `
+  -Name "DSCC-API-Vol" `
+  -SizeMib 16384 `
+  -SnapCpg "SSD_r6" `
+  -UserCpg "SSD_r6"  
+try {
+	$Result = Invoke-VolumeCreate -SystemId $SystemId -CreateVolumeInput $CreateVolumeInput
+} catch {
+    Write-Host ("Exception occurred when calling Invoke-CreateVolume: {0}" -f ($_.ErrorDetails | ConvertFrom-Json))
+    Write-Host ("Response headers: {0}" -f ($_.Exception.Response.Headers | ConvertTo-Json))	
+}
+```
+
+T﻿he *Invoke-VolumeCreate* command will return the information of the generated task. You can use the *Get-Task* command to monitor the the progress of this task.
```

Comment:
There are two "the"'s in the second sentence. Please remove one of them (just prior to "progress")